### ISSUE TITLE
[VESSEL ISSUE]  emolt_issues line 50

### ISSUE THREAD

**Issue by GMaynard1** [GMaynard1/emolt_project_management#6]
### Vessel Name

Lisa_Ann_III

### Issue Type

multiple_hauls

### Date Observed

2021-08-20

### Symptom

- probe unloads only when it returns to port
- not sure why but possibly cell phone stick

**Comment by xhx509** [comment]
Right, There are two possibilities, either the GPS puck issue or the logger range issue. Usually, it's the second one, The Raspberry pi keeps connecting to the logger, but failed in previous multiple attempts. when the boat returned to port, the downloading was successful, then we see the multiple_hauls in one file and in one telemetry transmission.
